### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by pmeinhardt**

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ config.plugin('copy').use(CopyWebpackPlugin, [
     patterns: [
       {
         from: src('icons', '*.png'),
-        to: '[name].[ext]',
+        to: '[name][ext]',
```
Comment (pmeinhardt) [inline webpack.config.babel.js:145]: https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/CHANGELOG.md#800-2021-03-04

**Comment by pmeinhardt** [comment]
Waddup Circle? 🤔

**Comment by pmeinhardt** [comment]
Bumping the Circle CI cache key seems to not do the trick. Can somebody check this out locally, `yarn install` and `yarn test` to see whether it's an issue on my machine?

**Comment by nickgnd** [comment]
@pmeinhardt I can reproduce the issue on my macbook

image

Apparently `commit` is undefined

> TypeError: Cannot read property 'commit' of undefined

And I guess it is due to this package `tickety-tick-formatter` that does not have any named export `defaults`

https://github.com/bitcrowd/tickety-tick-formatter/blob/main/src/index.ts

**Comment by nickgnd** [comment]
@pmeinhardt I pushed a commit, it should fix the issue 😉

**Comment by pmeinhardt** [comment]
> @pmeinhardt I can reproduce the issue on my macbook […] and I guess it is due to this package `tickety-tick-formatter` that does not have any named export `defaults`

Argh. Thanks @nickgnd. Something seems to be off with my local package installation then and I did not have the time this morning to properly investigate. 🙄 

Thanks a ton for the spot + fix. 💖

**Review by nickgnd — APPROVED** [review]
👍 Nice!

Thanks a ton 🙇

**Comment by pmeinhardt** [comment]
@tessi even if we install `tickety-tick-formatter` from GitHub, it might be nice to tag specific commits with a semver version number so it's clear whether the public API has changed in the meantime or not.

In this case, the `defaults` export had changed to `templateDefaults`:
https://github.com/bitcrowd/tickety-tick-formatter/commit/b5c5b38f1f29ed5c839fba76baffc3589b526eac

**Pull request merged**